modules/rosa-hcp-byo-oidc.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly.adoc
+// * rosa_getting_started/quickstart.adoc
+
+:_content-type: PROCEDURE
+[id="rosa-hcp-byo-oidc_{context}"]
+= Configuring your own OpenID Connect provider
+
+You can use your own Open ID Connect (OIDC) provider with {hcp} for {product title} (ROSA).
```

Comment:
The rendered version shows `{product title} (ROSA)` instead of the actual product name.